workbench/src/main/setupInvestHandlers.js

```diff
@@ -94,6 +94,7 @@ export function setupInvestRunHandlers() {
       cmdArgs = [
         'run',
         `--prefix ${settingsStore.get(`plugins.${modelRunName}.env`)}`,
+        '--live-stream',
```

Comment:
Cool, it looks like the mamba command was then capturing standard out/error without the `--no-capture-output, --live-stream` flag.